pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -282,6 +316,25 @@ func waitLoadbalancerDeleted(client *gophercloud.ServiceClient, loadbalancerID s
 	}
 }
 
+func (lbaas *LbaasV2) createNodeSecurityGroup(port int, protocol string, lbSecGroup string) error {
+	nodeSecGroupRuleCreateOpts := rules.CreateOpts{
+		Direction:     "ingress",
+		PortRangeMax:  port,
+		PortRangeMin:  port,
+		Protocol:      strings.ToLower(protocol),
+		RemoteGroupID: lbSecGroup,
+		SecGroupID:    lbaas.opts.NodeSecurityGroupID,
+		EtherType:     "IPv4",
```

Comment:
Please create a duplicate rule to allow IPv6 too.  We make no effort to filter to only IPv4 addresses in `NodeAddresses` and it's quite possible that traffic goes IPv4 to the LB and then IPv6 to the backend node.